src/rules/noUseBeforeDeclareRule.ts

```diff
@@ -42,14 +42,13 @@ class NoUseBeforeDeclareWalker extends Lint.ScopeAwareRuleWalker<VisitedVariable
     }
 
     public visitImportDeclaration(node: ts.ImportDeclaration) {
-        if (node.importClause != null) {
-            const importClause = node.importClause;
+        const importClause = node.importClause;
 
-            // named imports & namespace imports handled by other walker methods
-            if (importClause.name != null) {
-                const variableIdentifier = importClause.name;
-                this.validateUsageForVariable(variableIdentifier.text, variableIdentifier.getStart());
-            }
+        // Named imports & namespace imports handled by other walker methods.
+        // importClause will be null for bare imports.
+        if (importClause != null && importClause.name != null) {
+            const variableIdentifier = importClause.name;
```

Comment:
Doesn't matter though. I'm strongly in favor of using the strictest specifier whenever possible for variables. It signals to the developer that the variable binding should never change, and devs are expected to know JS/TS semantics enough to know that the values inside that object might change.